packages/contracts-bedrock/scripts/fpac/Makefile

```diff
@@ -0,0 +1,42 @@
+# ENV
+monorepo_root := "../../../.."
+deploy_config_path := "../../deploy-config"
+tmp := $(shell mktemp)
+
+# Help menu
+.PHONY: help
+help:
+	@echo "Available commands:"
+	@echo "  make cannon-prestate chain=<chain>"
+	@echo "  make deploy-fresh chain=<chain> args=<args>"
+	@echo "  make upgrade-game-impl chain=<chain> dgf=<dgf> vm=<vm> args=<args>"
+	@echo "  make update-init-bond chain=<chain> dgf=<dgf> game-type=<game-type> new-init-bond=<new-init-bond> args=<args>"
+
+# Generate the cannon prestate, and tar the `op-program` + `cannon` binaries + prestate data used to generate it.
+.PHONY: cannon-prestate
+cannon-prestate:
+	cd $(monorepo_root) && make cannon-prestate
+	@mkdir -p prestate-artifacts
+	@cp -r $(monorepo_root)/cannon/bin/** prestate-artifacts/
+	@cp -r $(monorepo_root)/op-program/bin/** prestate-artifacts/
+	tar -czvf prestate-artifacts.tar.gz prestate-artifacts
+	@jq ".faultGameAbsolutePrestate = $$(cat prestate-artifacts/prestate-proof.json | jq .pre)" "$(deploy_config_path)/$(chain).json" > $(tmp) && mv $(tmp) "$(deploy_config_path)/$(chain).json"
+	@echo "-------------------------------------------------------------------------------------------------------------"
+	@echo "-> Archive of prestate artifacts available at prestate-artifacts.tar.gz"
+	@echo "-> Set the absolute prestate to $$(cat prestate-artifacts/prestate-proof.json | jq .pre) in the $(chain) deploy config."
+	@rm -rf prestate-artifacts
```

Comment:
We're gonna need to change this to the reproducible method of generating the absolute prestate following https://github.com/ethereum-optimism/optimism/pull/8885